### PR TITLE
doc add make install

### PR DESCRIPTION
**Pull request by dlyr**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

* **Other information**:
Just one line of doc after user feedback.

### CODE REVIEW

File: doc/basics/compilation.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ $ cmake ..
 $ make
 ~~~
 
+Finally, the install target will copy all the radium related library in the same place, usefull for App compilation 
+
+~~~bash
+$ make install
+~~~
+
 To run on Mac OS X, one may need to set `DYLD_LIBRARY_PATH` to `path/to/Bundle-*/3rdPartyLibraries/lib`
```
Comment (MathiasPaulin) [inline doc/basics/compilation.md]: to be removed. No more needed.

Comment (dlyr) [reply doc/basics/compilation.md]: Could you provide a suggestion, because it not clear what I should remove

Comment (dlyr) [reply doc/basics/compilation.md]:
```suggestion
```

Comment (dlyr) [reply doc/basics/compilation.md]: Or just confirm thiss one ;) I will commit if its ok.

Comment (MathiasPaulin) [reply doc/basics/compilation.md]: OK, you can confirm. I've not the permission to push to the repo.

**Comment by dlyr** [comment]
@MathiasPaulin @nmellado do some can review the PR ?

**Comment by MathiasPaulin** [comment]
@dlyr approved. You can merge.

**Pull request merged**